ocs_ci/helpers/helpers.py

```diff
@@ -756,6 +756,9 @@ def create_storage_class(
             sc_data["parameters"]["encryptionKMSID"] = (
                 encryption_kms_id if encryption_kms_id else get_encryption_kmsid()[0]
             )
+        else:
```

Comment:
Can we move this else with outer if block?

Reply:
Done